charts/rucio-ui/templates/service.yaml

```diff
@@ -13,16 +13,28 @@ metadata:
 {{ toYaml . | indent 4 }}
 {{- end }}
 spec:
-  type: {{ $.Values.service.type }}
+  type: {{ .Values.service.type }}
   ports:
-    - port: {{ ternary 443 80 $.Values.service.useSSL }}
-      targetPort: {{ ternary "https" "http" $.Values.service.useSSL }}
-      protocol: TCP
-      {{- if $.Values.service.nodePort }}
-      nodePort: {{ $.Values.service.nodePort }}
-      {{- end }}
-      name: {{ ternary "https" "http" $.Values.service.useSSL }}
+    - port: {{ .Values.service.port }}
+      targetPort: {{ .Values.service.targetPort }}
+      protocol: {{ .Values.service.protocol }}
+      name: {{ .Values.service.name }}
   selector:
     app: {{ template "rucio.name" . }}
     release: {{ .Release.Name }}
+{{- if .Values.service.nodePort }}
+  nodePort: {{ .Values.service.nodePort }}
+{{- end }}
+{{- if .Values.service.loadBalancerIP }}
+  loadBalancerIP: {{ .Values.service.loadBalancerIP }}
+{{- end }}
+{{- if .Values.service.loadBalancerClass }}
+  loadBalancerClass: {{ .Values.service.loadBalancerClass }}
+{{- end }}
+{{- if .Values.service.externalTrafficPolicy }}
+  externalTrafficPolicy: {{ .Values.service.externalTrafficPolicy }}
+{{- end }}
+{{- if not .Values.service.allocateLoadBalancerNodePorts }}
+  allocateLoadBalancerNodePorts: {{ .Values.service.allocateLoadBalancerNodePorts }}
```

Comment:
should this be `if` instead of `if not`?

Reply:
good catch ;)